es6-test/unit/merger.js

```diff
@@ -232,8 +219,8 @@ describe('merging feature streams to layers', function() {
             "the_geom": {
               "type": "LineString",
               "coordinates": [
-                [-97.48784799692679, 0],
-                [-97.48783903791886, 0.000009019385540221545]
+                [100, 0],
```

Comment:
Also I'm a bit lost on why we're removing the crs from the fixture data on a PR where the handling of it is changed?